### PR TITLE
92/Metro dr fix

### PR DESCRIPTION
**Pull request by rawstorage**
ci will be switched over to this branch for few runs to ensure nothing is broken, small fix and added option to metro dr

### CODE REVIEW

**Comment by MichaelMcAleer** [comment]
Fixes issue https://github.com/dell/PyU4V/issues/105

**Comment by MichaelMcAleer** [comment]
CI test fixes have been commited to this pull request to save another update later. Tests now account for scenarios of 8 or 9 health checks and also where IP interfaces may not be configured in an environment.

**Review by MichaelMcAleer — CHANGES_REQUESTED** [review]
Functional code looks fine, needs associated unit tests and CI tests before approval

**Comment by MichaelMcAleer** [comment]
Pull request has been updated to include fix for default REST header application type.

This pull request makes the following changes:
- Fix for metro_dr delete function
- Fix for metro_dr added recover option
- Fix for CI tests:
    - PyU4V.tests.ci_tests.test_pyu4v_ci_system.CITestSystem._test_get_health_check_details
    - PyU4V.tests.ci_tests.test_pyu4v_ci_system.CITestSystem.test_get_ip_interface_list
    - PyU4V.tests.ci_tests.test_pyu4v_ci_system.CITestSystem.test_get_ip_interface
- Added Force Option to metro dr restore
- Application-Type header is now set by default to 'PyU4V-{version}'

**Review by helenwalsh — APPROVED** [review]

File: PyU4V/tests/unit_tests/test_pyu4v_metro_dr.py [hunk 1]
```diff
@@ -166,3 +166,11 @@ def test_modify_metrodr_environment_restore_both(self):
             exception.InvalidInputException,
             self.metro_dr.modify_metrodr_environment, action='restore',
             environment_name='PyU4V', metro=True, dr=True)
+
+    def test_modify_metrodr_environment_recover(self):
+        """Test create_metrodr_environment."""
+        with mock.patch.object(
+                self.metro_dr, 'modify_resource') as mock_put:
+            self.metro_dr.modify_metrodr_environment(action='recover',
+            environment_name='PyU4V')
```
Comment (MichaelMcAleer) [inline PyU4V/tests/unit_tests/test_pyu4v_metro_dr.py]: Missing indent, pep8 will fail

**Review by MichaelMcAleer — CHANGES_REQUESTED** [review]
Just the one fix needed Paul and it'll just make pep8 run clean. If CI runs cleanly over the weekend ill +1 on Monday, everything else looks fine.

**Review by MichaelMcAleer — APPROVED** [review]

**Comment by MichaelMcAleer** [comment]
CI has run cleanly, merging

**Pull request merged**